charts/gardener/controlplane/charts/application/templates/validatingwebhook-admission-controller.yaml

```diff
@@ -247,4 +247,32 @@ webhooks:
     {{- end }}
     caBundle: {{ required ".Values.global.admission.config.server.webhooks.tls.caBundle is required" (b64enc .Values.global.admission.config.server.webhooks.tls.caBundle) }}
   sideEffects: None
+- name: admission-plugin-secret.gardener.cloud
```

Comment:
Why is the validation not a native [admission plugin](https://github.com/gardener/gardener/tree/master/plugin/pkg) but a validatingwebhook? It only reacts on `shoot`events.

Reply:
/unhold 
@timuthy It was a miss from my side, webhook should react on secret update also so that users can't remove kubeconfig from secret if it is referenced by some shoot, I have reworked commit - 48a7175832f64433065a88694b81c7677561927a. PTAL